stubs/sso/__init__.py

```diff
@@ -148,12 +155,12 @@ def token_endpoint(tenant):
                 "nbf": int(time.time()),
                 "exp": int(time.time() + 60 * 60),
                 "idp": "",
-                "name": "Dev Developer",
+                "name": _name_from_email(email_address),
                 "nonce": dummy_nonce,
                 "oid": "",
                 "preferred_username": email_address,
                 "rh": "",
-                "roles": ["FSD_ADMIN"],
+                "roles": ["FSD_ADMIN" if email_address in app.config["FSD_ADMIN_LIST"] else "MEMBER"],
```

Comment:
`MEMBER` is not something that will be present in AD or come back from SSO. We only need to check for FSD_ADMIN 👍